Sources/SymbolKit/SymbolGraph/Symbol/KindIdentifier.swift

```diff
@@ -23,6 +23,8 @@ extension SymbolGraph.Symbol {
         case `func`
         case `operator`
         case `init`
+        case ivar
```

Comment:
We decided to leave this as an enum for now to reduce risk, right? Do we have a bug tracking converting this to a struct? Might be a good first-issue now that Swift-DocC has the `default` case so we don't expect any breakage.

Reply:
Basically, yeah. The conversion is in this PR: https://github.com/apple/swift-docc-symbolkit/pull/7

Reply:
Oh nice 👍🏻 Sounds good then.